website/addons/dryad/model.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+
+from modularodm import fields
+
+from website.addons.dryad import utils as dryad_utils
+from website.addons.base import AddonUserSettingsBase, AddonNodeSettingsBase
+
+
+class AddonDryadUserSettings(AddonUserSettingsBase):
```

Comment:
Minor, but preferred convention is `<addon_name>UserSettings`, rather than `Addon<addon_name>UserSettings`. Same goes for node